src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertIntegrationTest.java

```diff
@@ -1309,6 +1309,42 @@ private void _verifyInsert(String docId, JsonNode doc) {
 
   @Nested
   @Order(4)
+  class InsertOneFails {
```

Comment:
Separate IT beyond unit tests to ensure we actually have integration working end-to-end.